src/features/home/screen/index.tsx

```diff
@@ -59,16 +29,16 @@ const HomeScreen: FunctionComponent<HomePropTypes> = ({ navigation: { navigate }
     }, []),
   );
 
+  const { topics } = useGetTopics();
+
   return (
     <SafeAreaView sx={styles.container}>
       <ScrollView style={styles.scrollView}>
         <Text accessibilityRole="text" variant="h4" sx={styles.title}>
           {strings.HOME.title}
         </Text>
 
-        {TOPICS.map(topic => (
-          <Topic key={topic.name} topic={topic} />
-        ))}
+        {!!topics.length && topics.map(topic => <Topic key={topic.name} topic={topic} />)}
```

Comment:
```suggestion
        {!!topics.length && topics.map(topic => <Topic key={topic.id} topic={topic} />)}
```

I think you can use the id

Reply:
Yeah, I forgot to update. Thanks